### DISCUSSION TITLE
Migration du site sur le DSFR

### DISCUSSION THREAD

**Discussion by carolineBda** [SocialGouv/code-du-travail-numerique#5968] [Technique]
Objet : Passer l’ensemble du site sur le [Système de Design de l'État](https://www.systeme-de-design.gouv.fr/). Cette estimation n'inclus aucun autre changement que la migration du style.

| Complexité|Taille|
|--------|--------|
| -|3 jours|
| 🟠 |10 jours|
| 🔴  |20 jours |

### Pages spéciales 
- [Home](https://code.travail.gouv.fr/) 🔴 *dépend du design*
- [Droit du Travail](https://code.travail.gouv.fr/droit-du-travail) 🔴 ++
- [Dossiers](https://code.travail.gouv.fr/dossiers/aides-et-accompagnement-embauche-et-perennisation-des-emplois)

### Contenus éditoriaux
- [Fiche Ministère du travail](https://code.travail.gouv.fr/fiche-ministere-travail/epargne-salariale-partage-de-la-valeur-principes-generaux) 🟠
- [Fiche Service Public](https://code.travail.gouv.fr/fiche-service-public/interessement) 🔴 ++
- [Pages informations classiques](https://code.travail.gouv.fr/information/acquisition-de-conges-payes-pendant-un-arret-maladie-les-nouvelles-regles) 🔴
- [Pages informations dossier de licenciement](https://code.travail.gouv.fr/information/grand-licenciement-collectif-au-moins-10-salaries-pour-motif-economique-dans-une-entreprise-de-50-salaries-et-plus-avec-cse-le-contrat-de-securisation-professionnelle-csp) 🔴
- [Convention collective](https://code.travail.gouv.fr/convention-collective/2941-aide-accompagnement-soins-et-services-a-domicile-bad)
- [Contributions Génériques](https://code.travail.gouv.fr/contribution/la-periode-dessai-peut-elle-etre-renouvelee) 🟠
- [Contributions personnalisées](https://code.travail.gouv.fr/contribution/2941-la-periode-dessai-peut-elle-etre-renouvelee) 🟠
- [Modèles de documents](https://code.travail.gouv.fr/modeles-de-courriers/document-dinformation-sur-le-conge-de-reclassement) 🟠

### Pages simples
- [Recherche](https://code.travail.gouv.fr/recherche?q=r%C3%A9serve+sp%C3%A9ciale+de+participation) 🟠
- [Thèmes intermédiaires](https://code.travail.gouv.fr/themes/salaire-et-remuneration)
- [Glossaire - liste](https://code.travail.gouv.fr/glossaire)
- [Glossaire - définition](https://code.travail.gouv.fr/glossaire/a-titre-conservatoire)
- [À Propos](https://code.travail.gouv.fr/a-propos)
- [Stats](https://code.travail.gouv.fr/stats)
- [Mentions légales](https://code.travail.gouv.fr/mentions-legales)
- [Accessibilité](https://code.travail.gouv.fr/accessibilite)
- [Politique de confidentialité](https://code.travail.gouv.fr/politique-confidentialite)
- [Plan du site](https://code.travail.gouv.fr/plan-du-site)
- Nous contacter
- Accessibilité (modal pour les couleurs)

### Pages listes
- [Boîte à outils](https://code.travail.gouv.fr/outils) 🟠
- [Modèles de documents](https://code.travail.gouv.fr/modeles-de-courriers)
- [Vos fiches pratiques](https://code.travail.gouv.fr/contribution)
- [Votre convention collective](https://code.travail.gouv.fr/convention-collective)
- [Thèmes](https://code.travail.gouv.fr/themes) 🟠
- [Intégrer les outils du Code du travail numérique](https://code.travail.gouv.fr/integration) 🟠

### Outils
- [Indemnité de licenciement](https://code.travail.gouv.fr/outils/indemnite-licenciement) 🔴
- [indemnité de rupture conventionnelle](https://code-du-travail-numerique-preprod.ovh.fabrique.social.gouv.fr/outils/indemnite-rupture-conventionnelle) 🟠
- [Préavis de démission](https://code.travail.gouv.fr/outils/preavis-demission) 🔴
- [Salaire brut/net](https://code.travail.gouv.fr/outils/simulateur-embauche) 
- [Indemnité de précarité](https://code.travail.gouv.fr/outils/indemnite-precarite) 🔴
- [Préavis de licenciement](https://code.travail.gouv.fr/outils/preavis-licenciement) 🔴
- [Heures d'absence pour rechercher un emploi](https://code.travail.gouv.fr/outils/heures-recherche-emploi) 🔴
- [Trouver sa convention collective](https://code.travail.gouv.fr/outils/convention-collective) 🔴
- [Préavis de départ ou de mise à la retraite](https://code.travail.gouv.fr/outils/preavis-retraite) 🔴
- [Comprendre sa procédure de licenciement](https://code.travail.gouv.fr/outils/procedure-licenciement) 🟠
- [Page installation widgets](https://code.travail.gouv.fr/integration/moteur-recherche) 🟠

## Taches
- mise en place du DSFR sur le site en cohabitation avec le style actuel - *20 jours*
- migration des parties communes : header, footer, tooltips, articles liés, fil d'ariane... - *20 jours*
- migration des pages : composants, tests, clean du code
- Étapes de validation, SIG, RGAA ??

||Nombre|Total jours|
|--------|--------|--------|
| Simple|17|51 |
| 🟠 |11|110 |
| 🔴  |12 |240|

**Total 441 jours**

**Comment by carolineBda** [comment]
TODO : 
- affiner l'estimation des outils

**Comment by maxgfr** [comment]
Migration terminée 🎉
